packages/ckeditor5-engine/src/model/document.js

```diff
@@ -179,6 +152,20 @@ export default class Document {
 		} );
 	}
 
+	/**
+	 * The document version. Every operation increases the version number. It is used to ensure that
+	 * operations are applied on a proper document version.
+	 *
+	 * If the {@link module:engine/model/operation/operation~Operation#baseVersion base version} does not match the document version,
+	 * a {@link module:utils/ckeditorerror~CKEditorError model-document-applyoperation-wrong-version} error is thrown.
```

Comment:
```suggestion
	 * The document version.
	 *
	 * See {@link module:engine/model/history~History#version}.
```

Reply:
How about keeping it short and having doc in one place?

Reply:
Or we could go the other way around and keep the documentation here and make it simple in `History`. But I also see now that these docs differ a bit. This one says about the error, while `History` doc says about changing version property.